src/ai/backend/manager/api/vfolder.py

```diff
@@ -1858,7 +1858,7 @@ async def delete_from_trash_bin(
 async def force_delete(request: web.Request) -> web.Response:
     root_ctx: RootContext = request.app["_root.context"]
 
-    piece = request.match_info["name"]
+    piece = request.match_info["folder_id"]
     try:
         folder_id = uuid.UUID(piece)
```

Comment:
@fregataa I've renamed the parameter to `folder_id` to align with its purpose, as it's validated to be in a `UUID` format.